### PR TITLE
More EE Chunk Compat for Modded Ores

### PR DESCRIPTION
**Pull request by pm065**
Now only includes loot tables. I may have forgotten to remove some things, not sure.
I've backed up the chunk _additions_ in a new branch so that I can transfer them to my own modpack, which also utilizes EE, and in case you change your mind :D

### CODE REVIEW

**Comment by pm065** [comment]
Oh yeah! I forgot the dimensional mineshaft!

**Comment by pm065** [comment]
Fixed now

File: kubejs/data/mythicbotany/loot_tables/blocks/elementium_ore.json [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+          }
+        ],
+        "conditions": [
+          {
+            "condition": "minecraft:survives_explosion"
+          }
+        ]
+      }
+    ]
+  }
```
Comment (NillerMedDild) [inline kubejs/data/mythicbotany/loot_tables/blocks/elementium_ore.json:20]: Errr :P

File: kubejs/data/mythicbotany/loot_tables/blocks/elementium_ore.json [hunk 2]
```diff
@@ -0,0 +1,20 @@
+{
+    "type": "minecraft:block",
+    "pools": [
+      {
+        "name": "main",
+        "rolls": 1,
+        "entries": [
+          {
+            "type": "minecraft:item",
+            "name": "kubejs:elementium_chunk"
```
Comment (NillerMedDild) [inline kubejs/data/mythicbotany/loot_tables/blocks/elementium_ore.json:10]: Erhm :P

**Review by NillerMedDild — CHANGES_REQUESTED** [review]

File: kubejs/data/mythicbotany/loot_tables/blocks/elementium_ore.json (hunk 2 above)
Comment (pm065) [reply kubejs/data/mythicbotany/loot_tables/blocks/elementium_ore.json:10]: oops

**Comment by pm065** [comment]
resolved!

**Review by NillerMedDild — APPROVED** [review]

**Comment by NillerMedDild** [comment]
Thanks! :D

**Pull request merged**